src/options/views/tab-settings/index.vue

```diff
@@ -70,6 +60,21 @@
       </button>
     </div>
     <div v-show="showAdvanced">
+      <section>
+        <h3 v-text="i18n('labelGeneral')"></h3>
+        <div class="mb-1">
+          <label>
+            <span v-text="i18n('labelInjectionMode')"></span>
+            <select v-model="defaultInjectInto">
+              <option value="auto">auto</option>
+              <option value="page">page</option>
+              <option value="content">content</option>
+              <option value="wrap">wrap</option>
+            </select>
+            <a class="ml-1" href="https://violentmonkey.github.io/posts/inject-into-context/" target="_blank" rel="noopener noreferrer" v-text="i18n('learnInjectionMode')"></a>
```

Comment:
Now a short explanation hint near the selector is really needed because `wrap` is cryptic, and there is no substantially better name.

For example:

> `page` uses script elements in the page, `wrap` (Firefox only) same as `content` but unsafeWindow is mapped to the window object of the page,  `content` runs in the content script context to avoid CSP restrictions,  `auto` automatically uses one of these modes that succeeds in the listed order.

Reply:
Since there is no new concept here, maybe we can move the explanation to the linked documentation.

Reply:
Ideally I'd like to put all that info inside the extension and make it translatable :-)

Reply:
Could be something to do in the future though.